Revert to Bootstrap 3, take 2

kaapstorm opened the pull request:
![image](https://github.com/user-attachments/assets/72cae919-7d75-4ce7-ae54-60677b456a5f)

## Technical Summary

This pull request is similar to https://github.com/dimagi/commcare-hq/pull/36097 but instead of rebasing and then updating manually, I gave Claude 3.7 Sonnet the same prompt with the code in `master`. This is the result. Details given in the commit message.

## Feature Flag

campaign_dashboard

## Safety Assurance

### Safety story

Tested locally:
* Map filters
* Applying map filters
* Filter panel accordion
* Export to PDF
* Add widget

### Automated test coverage

No

### QA Plan

No

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

kaapstorm committed:
Just get Clause 3.7 Sonnet to do it all again

Here's a summary of what we changed:

1. Updated the base template from `hqwebapp/bootstrap5/base_section.html` to `hqwebapp/bootstrap3/base_section.html`.

2. Replaced Bootstrap 5 utility classes with Bootstrap 3 equivalents:
   - `d-flex`, `justify-content-between`, `align-items-center`, `mb-3` → `clearfix margin-bottom-10`, `pull-left`, `pull-right`
   - `d-none` → `hide`
   - `me-1` → `margin-right-5`
   - `mb-4` → `margin-bottom-20`
   - `p-3` → `padding-10`
   - `p-0` → `no-padding`
   - `g-0` → removed (no direct equivalent)
   - `border-end` → `border-right`

3. Updated nav/tab components:
   - Removed `.nav-item` and `.nav-link` classes
   - Changed `data-bs-toggle="tab"` to `data-toggle="tab"`

4. Updated modal components:
   - Changed `data-bs-dismiss="modal"` to `data-dismiss="modal"`
   - Changed `data-bs-toggle="modal"` to `data-toggle="modal"`
   - Changed `data-bs-target="#widget-modal"` to `data-target="#widget-modal"`
   - Replaced `btn-close` with `close` and added `<span aria-hidden="true">&times;</span>`
   - Moved modal title from `<h5>` to `<h4>` and repositioned it after the close button

5. Updated cards to panels:
   - `card` → `panel panel-default`
   - `card-header` → `panel-heading`
   - `card-body` → `panel-body`

6. Updated form components:
   - `form-check` → `checkbox`
   - Removed `form-check-label` class

7. Updated JavaScript references:
   - Changed import from 'reports/js/bootstrap5/base' to 'reports/js/bootstrap3/base'
   - Updated event handlers from `shown.bs.tab` to use Bootstrap 3's events
   - Updated selectors like `.nav-tabs .nav-link.active` to `.nav-tabs .active a`
   - Updated DOM manipulation methods for showing/hiding elements from `.classList.add('d-none')` to Bootstrap 3's `.addClass('hide')`

8. Removed all `use_bootstrap5` decorators from view functions

9. Updated include references from Bootstrap 5 to Bootstrap 3:
   - Changed `hqwebapp/htmx/error_modal.html` to `hqwebapp/bootstrap3/error_modal.html`
   - Changed `reports/standard/partials/bootstrap5/filter_panel.html` to `reports/standard/partials/bootstrap3/filter_panel.html`

mkangia commented:
Hey @kaapstorm 

We moving campaign back to bootstrap3!? Or I should say we need to move campaign down to bootstrap3?

kaapstorm committed:
`js_entry` to `js_entry_b3 "campaign/js/dashboard"`

I am not sure how testing succeeded without this?! Caching? Something
slow to restart?

kaapstorm commented:
> We moving campaign back to bootstrap3!?

I thought I discussed this in a group Slack, and now I realize it was only in a few DMs. :slightly_frowning_face: Sorry about that.

Yeah. Not all report filters are available in BS5. I don't think we have enough time to migrate the ones that aren't. (But I'd love to be wrong.) So I don't think we have a choice.

ajeety4 reviewed (commented):
Overall the code changes looks fine. 
Did a happy path testing and functionality is intact.

There is a minor UI alignment issue of headers and cards that we should fix.

ajeety4 commented:
> There is a minor UI alignment issue of headers and cards that we should fix.

Note. For some reason, the tab content (`<div class="tab-content">`) do not picks up padding from the hq container. So simply adding `padding-left: 10px` fixes the alignment issue.

zandre-eng reviewed (commented):
Took a pass through changes and overall everything looks good. Happy to approve once padding issue is resolved as mentioned in [Ajeet's comment](https://github.com/dimagi/commcare-hq/pull/36100#issuecomment-2765614725).

kaapstorm commented:
This change is not necessary. :sweat_smile:  Closing.

mkangia commented:
Thank god! :)
